### DISCUSSION TITLE
本地预览正常，部署后不正常

### DISCUSSION THREAD

**Discussion by weaving-dreams** [YunYouJun/hexo-theme-starry#264] [Q&A 疑难杂症] [answered]
本地预览正常https://s3.bmp.ovh/imgs/2022/03/cd2a57bf2a58bb28.bmp
部署后：https://weaving-dreams.github.io/weaving-dreams/

**Comment by YunYouJun** [comment, marked as answer]
使用了子目录时，需要在 `_config.yml` 中配置 `root: /你的子路径`。
